app/views/layouts/component_preview.html.erb

```diff
@@ -4,7 +4,7 @@
     <title>Component Preview</title>
     <%= stylesheet_link_tag 'application', media: 'all' %>
   </head>
-  <body class="padding-2">
+  <body class="padding-2 <%= params[:lookbook][:display][:body_class] %>">
```

Comment:
```suggestion
  <body class="padding-2 <%= params.dig(:lookbook, :display, :body_class) %>">
```
This is safer in case one of the intermediate hashes is nil

Reply:
> This is safer in case one of the intermediate hashes is nil

👍 Updated in dfac96022.